docs/UserGuide.md

```diff
@@ -63,6 +63,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* If you are using a PDF version of this document, be careful when copy-pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
```

Comment:
A minor alteration suggestion:
It may be less confusing to users if we adjust the wording regarding the copy-pasting commands. Instead of suggesting that AB3 has a copy-paste command, we could articulate it as 'copying and pasting the command from the PDF version of the User Guide.'